### DISCUSSION TITLE
Jump to PDF by clicking on the source

### DISCUSSION THREAD

**Discussion by yuxineverforever** [James-Yu/LaTeX-Workshop#3177] [Q&A] [answered]
## Preliminary questions [Required]

I am wondering if there is any shortcut that pdf can jump to specific location with user is refering to the source code.

I see click+right on the pdf can jump to the source. I just want the reverse operation.

Again, much appreciate for the efforts made by all the developers! This extension is lightweight, fast and user-friendly! @James-Yu @joaomoreno @criemen @simon-nicholls @erickguan @chatziko

**Comment by jlelong** [comment, marked as answer]
If I understand well, you are talking about forward `synctex`: you want to jump to the PDF from the `.tex` file. It is explained in the wiki: https://github.com/James-Yu/LaTeX-Workshop/wiki/View#usage

**Reply by yuxineverforever** [reply]
Thanks! It works!
